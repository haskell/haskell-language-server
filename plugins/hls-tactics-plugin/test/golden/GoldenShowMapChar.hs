test :: Show a => a -> (String -> b) -> b
test = _
