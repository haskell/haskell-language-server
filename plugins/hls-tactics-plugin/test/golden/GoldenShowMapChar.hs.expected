test :: Show a => a -> (String -> b) -> b
test a fl_cb = fl_cb (show a)
